docs/components/iam/deployment/configuration-variables.md

```diff
@@ -12,6 +12,13 @@ Environment variable | Description | Default value
 -----|-------------|--------------
 FEATURE_LDAP | Toggle LDAP support within IAM | false
 
+### Functionality
+
+Environment variable | Description | Default value
+-------|----------|-------
+ENFORCE_ACCESS_CONTROL | Controls the enforcing of permissions for the IAM component | false
```

Comment:
I would also add more details like "If set to `false`, all users can access user, roles and permission management."